packages/go/stbernard/command/license/license.go

```diff
@@ -0,0 +1,77 @@
+//  Copyright 2025 Specter Ops, Inc.
+//
+//  Licensed under the Apache License, Version 2.0
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+//  SPDX-License-Identifier: Apache-2.0
+//
+
+package license
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	license "github.com/specterops/bloodhound/packages/go/stbernard/command/license/internal"
+	"github.com/specterops/bloodhound/packages/go/stbernard/environment"
+)
+
+const (
+	Name  = "license"
+	Usage = "Run license cmd to append license headers on bhce files"
```

Comment:
Remove bhce from the naming, since the command isn't actually tied to bhce specifically